### ISSUE TITLE
Support for native authentication (feature request)

### ISSUE THREAD

**Issue by AdriVanHoudt** [nylas/nylas-nodejs#71]
https://docs.nylas.com/reference#native-authentication-1

Basically helper functions for the authorize and token calls.

**Comment by Gerdie** [comment]
Thanks for reporting @AdriVanHoudt we will be prioritizing this in the next quarter.

**Comment by smrutherford** [comment]
Hi @AdriVanHoudt, we've just released v4.8.0 of the NodeSDK which now includes helper functions for the authorize and token calls. Now, you may auth a user with Native Auth through the NodeSDK by doing the following:
```
Nylas.config({ clientId: CLIENT_ID, clientSecret: CLIENT_SECRET, });
Nylas.connect.authorize({name: 'test', email_address: emailAddress, provider: 'exchange', settings: settings, scopes: scopes});
Nylas.connect.token({{your_code}});
```
